scripts/src/java/org/oppia/android/scripts/ci/ComputeAffectedTests.kt

```diff
@@ -28,27 +29,28 @@ private const val MAX_TEST_COUNT_PER_SMALL_SHARD = 15
  * Arguments:
  * - path_to_directory_root: directory path to the root of the Oppia Android repository.
  * - path_to_output_file: path to the file in which the affected test targets will be printed.
- * - base_develop_branch_reference: the reference to the local develop branch that should be use.
- *     Generally, this is 'origin/develop'.
+ * - merge_base_commit: the base commit against which the local changes will be compared when
+ *     determining which tests to run. When running outside of CI you can use the result of running:
+ *     'git merge-base develop HEAD'
  * - compute_all_tests: whether to compute a list of all tests to run.
  *
  * Example:
  *   bazel run //scripts:compute_affected_tests -- $(pwd) /tmp/affected_test_buckets.proto64 \\
- *     origin/develop compute_all_tests=false
+ *     abcdef0123456789 compute_all_tests=false
  */
 fun main(args: Array<String>) {
   if (args.size < 4) {
     println(
       "Usage: bazel run //scripts:compute_affected_tests --" +
-        " <path_to_directory_root> <path_to_output_file> <base_develop_branch_reference>" +
+        " <path_to_directory_root> <path_to_output_file> <merge_base_commit>" +
```

Comment:
TODO: Document this change (that the merge base branch is now provided by the GitHub workflow).

Reply:
Done.